### ISSUE TITLE
Code for interacting with MLLM

### ISSUE THREAD

**Issue by Qrteng** [zhenyuw16/GenArtist#3]
Thank you for presenting such awesome work! I noticed that the code provided seems to be a simplified version compared to what you described in the paper. Specifically, the automatic image generation and editing process conducted by MLLM as an agent are not present in the code, such as the decomposition step, tree generation, self-correction and editing.

Could you please clarify if these parts will be released in the future? If I have misunderstood something, please let me know. Thank you very much for your assistance!

**Comment by newcodevelop** [comment]
Yes, these things are not available. Could you kindly provide the code for the same?
